File: DAY_44/DAY_44_02.py
# Created by KUNTAL OJHA .
# Today 14/10/2022 (dd/mm/yyyy)
# Problem Link: https://www.codechef.com/submit/MARKSTW
# Problem Title: Alice and Marks
# Programmng Language: PYTHON



# 1st method
x,y=map(int,input().split())

if x>= 2*y:
    print("YES")
else:
    print("NO")



# 2nd method
def main():
    x,y=map(int,input().split())

    if x>= 2*y:
        print("YES")
    else:
        print("NO")
        
if __name__=='__main__':
    main()

File: DAY_44/DAY_44_02.cpp
// Created by KUNTAL OJHA .
// Today 14/10/2022 (dd/mm/yyyy)
// Problem Link: https://www.codechef.com/submit/ENSPACE
// Problem Title: Enough Space
// Programmng Language: C++



#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while(t--)
	{
	    int n,x,y;
	    cin>>n>>x>>y;
	    
	    if(n>=(x*1)+(y*2))
	    {
	        cout<<"YES"<<endl;
	    }else
	    {
	        cout<<"NO"<<endl;
	    }
	}
	return 0;
}
